docs/cicd/README.md

```diff
@@ -81,12 +81,14 @@ Then, modify your `cicd` section in the config file (`default.json`) and create/
       {
         "target": "staging",
         "withDomain": false,
-        "requireManualApproval": false
+        "requireManualApproval": false,
+        "privileged": true
       },
       {
         "target": "production",
         "withDomain": false,
-        "requireManualApproval": true
+        "requireManualApproval": true,
+        "privileged": true
```

Comment:
Just interested to know more what the privileged flag does?

Reply:
The `privileged` should be set to true to allow docker stuff in the pipeline.

Without it this command won't work:

```
docker build -t ${ECR_REPOSITORY_NAME} . 
```

It sets this value inAWS pipeline:
![Screenshot 2020-10-13 at 16 14 32](https://user-images.githubusercontent.com/2649145/96152941-60ebf500-0f0d-11eb-9d59-0c492b57fe0a.png)